pod2daemon/Makefile

```diff
@@ -30,6 +30,7 @@ clean:
 	rm -rf bin/flexvol-$(ARCH)
 	rm -rf bin/$(CSI_IMAGE)-$(ARCH)
 	rm -rf bin/$(REGISTRAR_IMAGE)-$(ARCH)
+	rm -rf bin
```

Comment:
Do we need to do the equivalent to easily clean out all of the FIPS images in the other Makefiles too?